modules/core/include/opencv2/core/mat.hpp

```diff
@@ -1970,9 +1970,11 @@ class CV_EXPORTS Mat
     //! pointer to the data
     uchar* data;
 
-    //! helper fields used in locateROI and adjustROI
+    //! beginning of (parent) matrix -> needed when submatrix
```

Comment:
In my opinion, the original comment is better. Because it explains the purpose of the filed. The end user does not need to care about this field.

Or it may be changed to:
> Internal use only. Used for method locateROI and adjustROI.